packages/print-styles/src/components/code.css

```
@@ -1,16 +1,16 @@
 @media print {
   pre {
-    border: 1px solid #ddd;
-    border-radius: 4px;
-    padding: 0.8em;
+    border: 1px solid var(--yaae-print-code-border-color);
+    border-radius: var(--yaae-print-code-border-radius);
+    padding: var(--yaae-print-code-padding);
     white-space: pre-wrap;
     word-break: break-word;
```

Stylelint (17.3.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)